### PR TITLE
Add note about rust-g for building on Linux

### PR DESCRIPTION
**Pull request by Cyprex**
## About The Pull Request

This PR adds a minor note about needing to compile rust-g to Linux building instructions. I'm not sure if this is the appropriate place for it, but I don't see a better place to put it.

### CODE REVIEW

File: tools/build/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ The script will skip build steps whose inputs have not changed since the last ru
 
 - On Windows, `BUILD.bat` will automatically install a private (vendored) copy of Node.
 - On Linux, install Node using your package manager or from <https://nodejs.org/en/download/>.
+- On Linux you will likely need to compile rust-g, for instructions see https://github.com/tgstation/rust-g
```
Comment (Cyprex) [inline tools/build/README.md:20]: Not sure if a comma is needed here as in: `On Linux, you....`

Comment (TheBonded) [reply tools/build/README.md:20]: Replace the comma you already have with a period, then put a comma after linux. Comma splicing kills!

Comment (TiviPlus) [reply tools/build/README.md:20]:
```suggestion
- On Linux you will need to compile rust-g, for instructions see https://github.com/tgstation/rust-g
```
its not packaged so it's garaunteed

Comment (spookydonut) [reply tools/build/README.md:20]: tgs has scripts for adding it I think?  @bobbahbrown @Cyberboss

Comment (Cyberboss) [reply tools/build/README.md:20]: mentioned in the hosting documentation

Comment (bobbahbrown) [reply tools/build/README.md:20]: PreCompile.sh for /tg/ and consequently tgmc has it

Comment (spookydonut) [reply tools/build/README.md:20]: @Cyprex change it to say that unless using tgs4 or later that you will need to build/obtain your own rust_g.so

Comment (Cyberboss) [reply tools/build/README.md:20]: No need to specify 4 or later, cause 1-3 are windows only :honk:

**Pull request merged**